project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject fullcontact/full.metrics "0.10.4-SNAPSHOT"
+(defproject fullcontact/full.metrics "0.11.0-SNAPSHOT"
```

Comment:
I'd like to wait on the 1.0.0, as I think this lib could use a stronger interface, thoughts?